opba-auth-rest-impl/src/main/java/de/adorsys/opba/tppauthapi/config/CookieProperties.java

```diff
@@ -0,0 +1,24 @@
+package de.adorsys.opba.tppauthapi.config;
+
+import lombok.Data;
+import lombok.ToString;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.convert.DurationUnit;
+import org.springframework.context.annotation.Configuration;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+@Data
+@Configuration
+@ToString
+@ConfigurationProperties(prefix = "cookie")
+public class CookieProperties {
+
+    private boolean secure = true;
+    @DurationUnit(ChronoUnit.SECONDS)
```

Comment:
This is not necessary - `@DurationUnit`, we should write duration as PT10S for 10 seconds (PT10S is ISO-8601 standartized duration)

Reply:
done